codersdk/organizations.go

```diff
@@ -61,6 +61,15 @@ type CreateTemplateRequest struct {
 	// templates, but it doesn't make sense for users.
 	VersionID       uuid.UUID                `json:"template_version_id" validate:"required"`
 	ParameterValues []CreateParameterRequest `json:"parameter_values,omitempty"`
+
+	// MaxTTLMillis allows optionally specifying the maximum allowable TTL
+	// for all workspaces created from this template.
+	MaxTTLMillis *int64 `json:"max_ttl_ms,omitempty"`
+
+	// MinAutostartIntervalMillis allows optionally specifying the minimum
+	// allowable duration between autostarts for all workspaces created from
+	// this template.
+	MinAutostartIntervalMillis *int64 `json:"min_autostart_interval_ms,omitempty"`
```

Comment:
Making these optional in creation; if preferred I can make them mandatory.

Reply:
Is this so you don't have some autostart config that says start my workspace every minute?

Reply:
Yup!

Reply:
Thought: Can we make he zero values useful here? Or can these be set to zero to disable them entirely? If zero doesn't need to disable, we can treat it as not set, avoiding the use of pointers.

Reply:
FE prefers to consume null instead of zero-values; hence the pointers.
It would be nice though; I'll open a follow-up PR to investigate this.